src/pybind/mgr/cephadm/module.py

```diff
@@ -1047,6 +1066,10 @@ def _get_container_image(self, daemon_name: str) -> Optional[str]:
             image = self.container_image_alertmanager
         elif daemon_type == 'node-exporter':
             image = self.container_image_node_exporter
+        elif daemon_type == 'haproxy':
```

Comment:
is `haproxy` a generic haproxy type or crafted for RGW?

Reply:
this is crefated for rgw.... and I suggest to keep it in this way until we decide to introduce more services using this HA approach.